### PR TITLE
docs: overview draft

### LINKED ISSUE #1072: Draft documentation
Finalize all the draft documentation

### PR DESCRIPTION
**Pull request by clearab**
More doc changes for the overview page. fixes #1072

### CODE REVIEW

File: docs/overview.md [hunk 1]
```diff
@@ -25,14 +25,14 @@ Typically, your component will be made up of the following pieces:
 - Your component code
 - An implementation of the 'BotComponent' class to register your components with the runtime
   - You'll also need to update the `components` array in the `appsettings.json` file with your component name
-- A schema file that declares the inputs and outputs your component requires
-- A `uischema` file to tell Composer where to display your components
+- One or more `.schema` files that declares the inputs and outputs your component requires
+- One or more `.uischema` files to tell Composer where to display your components
```
Comment (tracyboehrer) [inline docs/overview.md:29]: uischema is optional, right?  But recommended?

File: docs/overview.md [hunk 2]
```diff
@@ -25,14 +25,14 @@ Typically, your component will be made up of the following pieces:
 - Your component code
 - An implementation of the 'BotComponent' class to register your components with the runtime
   - You'll also need to update the `components` array in the `appsettings.json` file with your component name
-- A schema file that declares the inputs and outputs your component requires
-- A `uischema` file to tell Composer where to display your components
+- One or more `.schema` files that declares the inputs and outputs your component requires
```
Comment (tracyboehrer) [inline docs/overview.md:28]: schema files just for BotComponents?  Like it's not needed for declarative packages only?

**Review by tracyboehrer — CHANGES_REQUESTED** [review]
Also, shouldn't we document more than just requiring the 'msbot-component' since Composer needs to understand more about certain types?

File: docs/overview.md (hunk 1 above)
Comment (clearab) [reply docs/overview.md:29]: This is just the overview, we'd go into all of those details in the "how to create a package" article.

File: docs/overview.md (hunk 2 above)
Comment (clearab) [reply docs/overview.md:28]: Same as above, but also that why I mention components in this line

**Comment by thomas-chung** [comment]
Hey @clearab is this PR still active?

**Comment by clearab** [comment]
My bad

**Comment by thomas-chung** [comment]
Naw it's all cool!

**Pull request closed without merging**